rootfs/entrypoint.sh

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/dumb-init /bin/sh
 source version.sh
-
+if [ -n "${SSH_KEY}" ]; then
+    echo "$SSH_KEY" > /id_rsa/custom_rsa
+    chmod 0600 /id_rsa/custom_rsa
+    KEY_FILE=/id_rsa/custom_rsa
+    SSH_KEY_FILE=/id_rsa/custom_rsa
+else
+    KEY_FILE=${SSH_KEY_FILE:=/id_rsa}
+fi
```

Comment:
Your order is backwards here.  Since `SSH_KEY_FILE` overrides `SSH_KEY`, the logic has to flow differently.

Why set `KEY_FILE` twice? `KEY_FILE` is set to `SSH_KEY_FILE` if and only if it exists, use that logic.